Core/Utilities/ThreadPoolExecutor.py

```diff
@@ -0,0 +1,120 @@
+"""
+ThreadPoolExecutor implements the Pool abstarct interface and is using concurrent.futures.ThreadPoolExecutor.
+Note: You can not use the ThreadPoolExecutor with 'with' for example:
+   with ThreadPoolExecutor ( 2 ) as executor:
+     executor.submit()
+
+"""
+import sys
+gMissingLibrary = False
+try:
+  import concurrent.futures
+except ImportError:
+  from DIRAC.Core.Utilities.ThreadPool import ThreadPool
+  global gMissingLibrary
+  gMissingLibrary = True
```

Comment:
Please add a "FIXME" that this should be cleaned once `futures` is part of the Externals